Translation corrections (#433)
07th-mod/higurashi-console-arcs: Update/ura_009.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1764", 256, TRUE);
	OutputLine(NULL, "（そもそも、相手側の親族に案内状の筆耕を頼むかフツー？）",

> 			NULL, "(Besides, do you normally ask the bride's family to write the invitations?)", Line_Normal);

Reviewer: ```suggestion
			NULL, "(Besides, is it normal to ask the bride's family to write the invitations?)", Line_Normal);
```
